[2025-07-27T10:13:17.157+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-27T10:13:17.296+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-07-26T00:00:00+00:00 [queued]>
[2025-07-27T10:13:17.366+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-07-26T00:00:00+00:00 [queued]>
[2025-07-27T10:13:17.370+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-27T10:13:17.441+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): section_2.inner_section_2.task_2> on 2025-07-26 00:00:00+00:00
[2025-07-27T10:13:17.499+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=361) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-27T10:13:17.513+0000] {standard_task_runner.py:64} INFO - Started process 479 to run task
[2025-07-27T10:13:17.489+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_task_group', 'section_2.inner_section_2.task_2', 'scheduled__2025-07-26T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_task_group.py', '--cfg-path', '/tmp/tmp7fsojpyk']
[2025-07-27T10:13:17.519+0000] {standard_task_runner.py:91} INFO - Job 23: Subtask section_2.inner_section_2.task_2
[2025-07-27T10:13:17.603+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-27T10:13:18.155+0000] {task_command.py:426} INFO - Running <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-07-26T00:00:00+00:00 [running]> on host 4112a2618ff3
[2025-07-27T10:13:19.244+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_task_group' AIRFLOW_CTX_TASK_ID='section_2.inner_section_2.task_2' AIRFLOW_CTX_EXECUTION_DATE='2025-07-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-26T00:00:00+00:00'
[2025-07-27T10:13:19.269+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-27T10:13:19.296+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-27T10:13:19.315+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 1']
[2025-07-27T10:13:19.959+0000] {subprocess.py:86} INFO - Output:
[2025-07-27T10:13:20.065+0000] {subprocess.py:93} INFO - 1
[2025-07-27T10:13:20.082+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-27T10:13:20.115+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-27T10:13:20.315+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_task_group, task_id=section_2.inner_section_2.task_2, run_id=scheduled__2025-07-26T00:00:00+00:00, execution_date=20250726T000000, start_date=20250727T101317, end_date=20250727T101320
[2025-07-27T10:13:20.671+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-07-27T10:13:21.080+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-27T10:13:21.098+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
